make/libraries

```diff
@@ -33,6 +33,10 @@ $(CVODES)/lib/libsundials_nvecserial.a: $(SUNDIALS_NVECSERIAL)
 
 LIBCVODES = $(CVODES)/lib/libsundials_nvecserial.a $(CVODES)/lib/libsundials_cvodes.a
 
+STAN_CVODES_HEADERS := $(shell find stan -name *cvodes*.hpp)
```

Comment:
Does this work on Windows?
(Also generally curious if we 1) support windows with these makefiles and 2) what our windows dependencies are. Is Rtools sufficient? What about for Python people?)